src/build/stable/utils/versions/get_version_from_package_json.ts

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'fs/promises';
+import { join } from 'path';
+
+import { AZLE_PACKAGE_PATH } from '../global_paths';
+
+export async function getVersionFromPackageJson(name: string): Promise<string> {
```

Comment:
I think we should rename this to getAzleVersionFromPackageJson

Reply:
Or just getAzleVersion...but there is a MUCH simpler way to do this

Reply:
done